### ISSUE TITLE
Restic Swift Error - "TooManyRequests"

### ISSUE THREAD

**Issue by sashokbg** [restic/restic#3739]
Output of `restic version`
--------------------------
restic 0.13.1 compiled with go1.18.1 on linux/amd64

How did you run restic exactly?
-------------------------------
I ran my "openrc.sh" to authenticate with Open Stack Swift.

My "env | grep OS"

```
OS_AUTH_URL=https://auth.cloud.ovh.net/v3
OS_IDENTITY_API_VERSION=3
OS_USER_DOMAIN_NAME=Default
OS_PROJECT_DOMAIN_NAME=Default
OS_TENANT_ID=REDACTED
OS_TENANT_NAME=REDACTED
OS_USERNAME=REDACTED
OS_PASSWORD=VERY REDACTED
OS_REGION_NAME=DE
```

. restic -r swift:home-cloud-backup:/backup1 backup cluster

```
Load(<key/e9ec344cac>, 0, 0) returned error, retrying after 552.330144ms: conn.ObjectOpen: TooManyRequests
Load(<key/e9ec344cac>, 0, 0) returned error, retrying after 1.080381816s: conn.ObjectOpen: TooManyRequests
Load(<key/e9ec344cac>, 0, 0) returned error, retrying after 1.31013006s: conn.ObjectOpen: TooManyRequests
```

Doing a simple "swift upload home-cloud-backup dir" works with no problems.

What backend/server/service did you use to store the repository?
----------------------------------------------------------------
Swift Cloud Archive on OVH

Expected behavior
-----------------
Upload the backup to the swift backend

Actual behavior
---------------
Restic managed to upload 2 files (see image below), so it looks like it is not authentication related.

![image](https://user-images.githubusercontent.com/10643824/166902980-48d09a67-b993-4667-b929-6b9a8e79b084.png)

Steps to reproduce the behavior
-------------------------------
- Create an OVH Account
- Create a user with "storage" permissions
- Download the openrc.sh file
- source openrc.sh -> enter password
- run restic init
- run restic backup

Do you have any idea what may have caused this?
-----------------------------------------------
Unfortunately no, but I think it might be related to running too many parallel connections.

Do you have an idea how to solve the issue?
-------------------------------------------
No

Did restic help you today? Did it make you happy in any way?
------------------------------------------------------------

Yes, restic and k8up would be the perfect solution form my home-cloud solution. It is open source and free as in freedom :) projects that make it possible to be independent from big enterprises and also learn a thing or two on the way ;)

**Comment by sashokbg** [comment]
Hello, I tried to enable debug logs as described in the tutorial https://restic.readthedocs.io/en/latest/090_participating.html
After building locally the problem seems to disappear and my backups are correctly uploaded.

Here is the output of my "./restic version" local:

restic 0.13.1 (v0.13.0-126-g26c33332) compiled with go1.18.1 on linux/amd64

What is strange is that the version seems to be the same (probably different build ?)

**Comment by MichaelEischer** [comment]
Getting a `TooManyRequests` error right when loading the repository keys is odd. At that point restic has barely issued any request at all to the backend. Are you able to still replicate the error using the regular restic 0.13.1 build? Otherwise I'm tempted to say that the backend did something strange...

**Comment by sashokbg** [comment]
Hello @MichaelEischer, this morning everything works fine with both official release and compiled version. I will contact my cloud provider (OVH) to see if the problem is related to their API quotas.

**Comment by sashokbg** [comment]
Hello, @MichaelEischer I am sorry I thought the issue was gone but for some reasone it appeared again with both versions.

I have found a page listing the limitations of the swift sotrage of my cloud provider: https://docs.ovh.com/us/en/storage/pcs/capabilities-and-limitations/

I have noticed that my files are actually frozen since I am using the "archive" option for this cloud. It is kind of like S3 glacier.

![image](https://user-images.githubusercontent.com/10643824/168164345-5d469cb4-d038-40ad-881e-604510cc86bf.png)

Maybe the TooManyRequests is comming from asking too many times the backend to unfreeze a file ?
